core/trino-main/src/main/java/io/trino/metadata/FunctionMetadata.java

```diff
@@ -22,60 +22,91 @@
 public class FunctionMetadata
 {
     private final FunctionId functionId;
-    private final Signature signature;
+    private final Signature canonicalSignature;
     private final boolean nullable;
     private final List<FunctionArgumentDefinition> argumentDefinitions;
     private final boolean hidden;
     private final boolean deterministic;
     private final String description;
     private final FunctionKind kind;
     private final boolean deprecated;
+    private final String actualName;
```

Comment:
Please place this field just after `canonicalSignature` field, since they are closely related.

Please apply same orddering in constructor parameters, constructor assignments, and getters

Reply:
copy that.